packages/components/src/components/expansion-panel/expansion-panel.jsx

```diff
@@ -39,20 +35,18 @@ class ExpansionPanel extends React.Component {
                             <div key={index} className='dc-expansion-panel__content-array'>
                                 <div
                                     className={classNames('dc-expansion-panel__content-array', {
-                                        'dc-expansion-panel__content-active': this.state.open_ids.includes(item.id),
+                                        'dc-expansion-panel__content-active': open_ids.includes(item.id),
                                     })}
                                 >
                                     <span className='dc-expansion-panel__content-array-item-index'>{index + 1}: </span>(
                                     {item.value.length})
                                     <Icon
                                         className='dc-expansion-panel__content-chevron-icon'
                                         icon='IcChevronRight'
-                                        onClick={() => this.onArrayItemClick(item.id)}
+                                        onClick={() => onArrayItemClick(item.id)}
                                     />
                                 </div>
-                                {this.state.open_ids.includes(item.id)
-                                    ? this.renderArrayContent(item.value.slice())
-                                    : null}
+                                {open_ids.includes(item.id) ? renderArrayContent(item.value.slice()) : null}
```

Comment:
You can remove the `slice` here since `renderArrayContent` isn't modifying the array (`.map` creates a new array).